--- util/Makefile.in.orig	2005-04-12 04:13:41.000000000 +0200
+++ util/Makefile.in	2005-09-17 16:13:18.000000000 +0200
@@ -143,7 +143,7 @@
 
 LDADD = ../lib/libcrack.la
 
-AM_CPPFLAGS = -I. -I.. -I$(top_srcdir)/lib '-DDEFAULT_CRACKLIB_DICT="$(pkgdatadir)/pw_dict"'
+AM_CPPFLAGS = -I. -I.. -I$(top_srcdir)/lib '-DDEFAULT_CRACKLIB_DICT="$(datadir)/dict/cracklib_dict"'
 
 cracklib_check_SOURCES = check.c
 cracklib_check_LDADD = $(LDADD)
--- lib/Makefile.in.orig	2005-04-12 04:13:38.000000000 +0200
+++ lib/Makefile.in	2005-09-17 16:14:08.000000000 +0200
@@ -152,7 +152,7 @@
 #
 libcrack_la_LDFLAGS = -version-info 10:0:8
 
-AM_CPPFLAGS = -I. -I.. -I$(top_srcdir)/lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="$(pkgdatadir)/pw_dict"'
+AM_CPPFLAGS = -I. -I.. -I$(top_srcdir)/lib -DIN_CRACKLIB '-DDEFAULT_CRACKLIB_DICT="$(datadir)/dict/cracklib_dict"'
 subdir = lib
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
