diff -durN -x '*~' cracklib,2.7.orig/cracklib/Makefile cracklib,2.7/cracklib/Makefile
--- cracklib,2.7.orig/cracklib/Makefile	2004-12-18 23:51:34.575367592 +0100
+++ cracklib,2.7/cracklib/Makefile	2004-12-18 23:52:52.645499128 +0100
@@ -11,6 +11,8 @@
 CFLAGS	= $(OPTFLAGS) -I../cracklib -DIN_CRACKLIB -fPIC
 LD	= ld
 
+libdir	= /usr/lib
+
 $(LIB):	$(OBJ) Makefile
 	$(LD) -shared -soname $(LIB).$(MAJOR) -o $(LIB).$(VERSION) $(OBJ) -lc
 	rm -f $(LIB).$(MAJOR) $(LIB)
@@ -21,6 +23,6 @@
 	-rm -f $(OBJ) $(LIB) $(LIB).$(VERSION) *~
 
 install: $(LIB) crack.h
-	install -m 755 $(LIB).$(VERSION) $(ROOT)/usr/lib
-	ln -sf $(LIB).$(VERSION) $(ROOT)/usr/lib/$(LIB)
+	install -m 755 $(LIB).$(VERSION) $(ROOT)$(libdir)
+	ln -sf $(LIB).$(VERSION) $(ROOT)$(libdir)/$(LIB)
 	install -m 644 crack.h $(ROOT)/usr/include
