diff -Nur cracklib,2.7/Makefile cracklib,2.7.pld/Makefile
--- cracklib,2.7/Makefile	Fri Jun  4 16:29:07 1999
+++ cracklib,2.7.pld/Makefile	Fri Jun  4 16:27:13 1999
@@ -16,12 +16,12 @@
 ###
 # set this to the absolute path (less extn) of compressed dict.
 
-DICTPATH="/usr/lib/cracklib_dict"
+DICTPATH="/usr/share/dict/cracklib_dict"
 
 ###
 # Set this to the path of one or more files continaing wordlists.
 
-SRCDICTS=/usr/dict/words /usr/dict/extra.words
+SRCDICTS=/usr/share/dict/words /usr/share/dict/extra.words
 
 ###
 # If you have installed the cracklib-dicts directory, use this
diff -Nur cracklib,2.7/cracklib/genrules.pl cracklib,2.7.pld/cracklib/genrules.pl
--- cracklib,2.7/cracklib/genrules.pl	Mon Dec 15 02:15:49 1997
+++ cracklib,2.7.pld/cracklib/genrules.pl	Fri Jun  4 16:28:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ###
 # This program is copyright Alec Muffett 1997. The author disclaims all
diff -Nur cracklib,2.7/util/create-cracklib-dict cracklib,2.7.pld/util/create-cracklib-dict
--- cracklib,2.7/util/create-cracklib-dict	Fri Jun  4 16:29:07 1999
+++ cracklib,2.7.pld/util/create-cracklib-dict	Fri Jun  4 16:27:58 1999
@@ -9,7 +9,7 @@
     echo "/usr/lib/cracklib_dict.*"
     echo
     echo "Example:"
-    echo "/usr/sbin/create-cracklib-dict /usr/dict/words"
+    echo "/usr/sbin/create-cracklib-dict /usr/share/dict/words"
 else
-    /usr/sbin/mkdict $* | /usr/sbin/packer /usr/lib/cracklib_dict
+    /usr/sbin/mkdict $* | /usr/sbin/packer /usr/share/dict/cracklib_dict
 fi
